packages/elements/src/components/app/app.component.ts

```diff
@@ -324,10 +324,18 @@ export class MutationTestReportAppComponent extends RealtimeElement {
         <div class="container bg-white pb-4 font-sans text-gray-800 motion-safe:transition-max-width">
           <div class="space-y-4 transition-colors">
             ${this.renderErrorMessage()}
-            <mte-theme-switch @theme-switch="${this.themeSwitch}" class="sticky top-offset z-20 float-right mx-4 pt-4" .theme="${this.theme}">
+            <mte-theme-switch @theme-switch="${this.themeSwitch}" class="sticky top-offset z-20 float-right pt-6" .theme="${this.theme}">
             </mte-theme-switch>
             ${this.renderTitle()} ${this.renderTabs()}
-            <mte-breadcrumb .view="${this.context.view}" class="my-4" .path="${this.context.path}"></mte-breadcrumb>
+            <mte-breadcrumb .view="${this.context.view}" .path="${this.context.path}"></mte-breadcrumb>
+            ${this.sse
+              ? html`<mte-progress-bar
+                  .killed="${this.rootModel?.systemUnderTestMetrics.metrics.killed}"
+                  .survived="${this.rootModel?.systemUnderTestMetrics.metrics.survived}"
+                  .pending="${this.rootModel?.systemUnderTestMetrics.metrics.pending}"
+                  .total="${this.rootModel?.systemUnderTestMetrics.metrics.totalMutants}"
```

Comment:
I think we're missing `NoCoverage` and `TimeOut`. Instead of using `killed` and `survived`, we should use `detected` and `undetected`. 

I'm also doubtful about the use of `totalMutants`, since they include `invalid` mutants. In practise, `totalMutants !== detected + undetected`. Won't that be a problem?

See https://stryker-mutator.io/docs/mutation-testing-elements/mutant-states-and-metrics/#metrics

Reply:
Should these also have different colors? Right now it has 4 different colors (green, yellow, red, grey)

Reply:
Do we not want invalid mutants in the progress bar?